examples/nlp/Tweet-classification-using-TFDF.py

```diff
@@ -0,0 +1,133 @@
+"""
+Title: FILLME
+Author: FILLME
+Date created: FILLME
+Last modified: FILLME
+Description: FILLME
+"""
+
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+import tensorflow_hub as hub
+from tensorflow.keras import layers
+import tensorflow_decision_forests as tfdf
+from sklearn.model_selection import train_test_split
+from tensorflow.keras.metrics import Recall, Precision, Accuracy, AUC
+
+# Turn .csv files into pandas DataFrame's
+train_df = pd.read_csv(
+    "https://raw.githubusercontent.com/IMvision12/Tweets-Classification-NLP/main/train.csv"
+    "https://raw.githubusercontent.com/IMvision12/Tweets-Classification-NLP/main/train.csv"
+)
+test_df = pd.read_csv(
+    "https://raw.githubusercontent.com/IMvision12/Tweets-Classification-NLP/main/test.csv"
+)
+train_df.head()
+
+train_df.shape
+
+train_df_shuffled = train_df.sample(frac=1, random_state=42)
+train_df_shuffled.drop(["id", "keyword", "location"], axis=1, inplace=True)
+train_df_shuffled.reset_index(inplace=True, drop=True)
+train_df_shuffled.head()
+
+train_df_shuffled.info()
+
+train_df_shuffled.target.value_counts()
+
+# How many samples total?
+print(f"Total training samples: {len(train_df)}")
+print(f"Total test samples: {len(test_df)}")
+print(f"Total samples: {len(train_df) + len(test_df)}")
+
+for ind, counter in enumerate(train_df_shuffled.index):
+    print(f"Target : {train_df_shuffled['target'][ind]}")
+    print(f"Text : {train_df_shuffled['text'][ind]}")
+    if counter == 5:
+        break
+
+# Use train_test_split to split training data into training and validation sets
+train_sentences, val_sentences, train_labels, val_labels = train_test_split(
+    train_df_shuffled["text"].to_numpy(),
+    train_df_shuffled["target"].to_numpy(),
+    test_size=0.2,  # dedicate 10% of samples to validation set
```

Comment:
(the comment doesn't match the code. But you should remove this code and do the split by hand)